docs/index.md

```diff
@@ -73,13 +73,18 @@ We expect everyone in our community (users, contributors, and employees alike) t
    install
    tutorial
    skaffold
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Guides
```

Comment:
I love the idea of Guides; I'd keep these in Getting Started because I think they're about onboarding, but it's certainly not blocking.

I want more Guides that aren't just for onboarding.

Reply:
My fear: a user does the Tutorial but it takes a while because they use X, then sees the article "X", clicks on it and goes "wow, I wish I'd known this when I was doing the Tutorial."

Therefore, articles that users wish they'd see before the Tutorial should be in "Getting started".

Which makes me think Helm and Docker Compose should be in there. (I also think that Example Projects is kinda in there, but if you don't have a project, you'll figure it out some way).

WDYT?

Reply:
SGTM! I'll move them back.